src/Microsoft.Health.Dicom.Core/Features/Common/DicomFileNameWithPrefix.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Features.Model;
+
+namespace Microsoft.Health.Dicom.Core.Features.Common;
```

Comment:
It makes sense to me for these files to be in the Blob project instead of Core, they are implementation details of blob storage.

Reply:
Moved to blob project